takwimu/takwimu_ui/src/Theme.js

```diff
@@ -5,11 +5,25 @@ const FONT_FAMILY_TEXT = '"Muli", sans-serif';
 
 const Theme = createMuiTheme({
   palette: {
-    primary: { main: '#29a87c' }
+    primary: { main: '#29a87c' },
+    analysis: {
+      main: '#f6fbfa' // ice-blue
+    }
```

Comment:
Just make this `iceBlue: #f6fbfa`
Its used here too https://github.com/TakwimuAfrica/Takwimu/pull/471/files#diff-d4778892734c83ea1c6e5bbed582942aR9

Reply:
Generalized it to `palette.info.main` @karimkawambwa